packages/dd-trace/test/heap_snapshots.spec.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+require('./setup/tap')
+
+const { mkdtempSync, readdirSync } = require('fs')
+const { tmpdir } = require('os')
+const { join } = require('path')
+const { start } = require('../src/heap_snapshots')
+
+const folder = mkdtempSync(join(tmpdir(), 'dd-trace-heap-snapshot-'))
```

Comment:
Not that it matters, but I assume the trailing `-` was a mistake?

```suggestion
const folder = mkdtempSync(join(tmpdir(), 'dd-trace-heap-snapshot'))
```